pkgs/development/tools/misc/sccache/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cargo, rustc, rustPlatform, pkgconfig, glib, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  version = "0.2.8";
+  name = "sccache-${version}";
+
+  src = fetchFromGitHub {
+    owner = "mozilla";
+    repo = "sccache";
+    rev = version;
+    sha256 = "08v8s24q6246mdjzl5lirqg0csxcmd17szmw4lw373hvq4xvf0yk";
+  };
+  cargoSha256 = "1lafzin92h1hb1hqmbrsxja44nj8mpbsxhwcjr6rf5yrclgwmcxj";
+  cargoBuildFlags = [ "--features=all" ];
+  nativeBuildInputs = [
+    pkgconfig cargo rustc
+  ];
+  buildInputs = [
+    openssl
+  ];
```

Comment:
To build on darwin (tested macos 10.13) this needs `++ stdenv.lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security` added to `buildInputs`

Reply:
Thanks for testing this! I'm adding this after the closing `]`. Please let me know if that's not what you meant.

Reply:
Perfect.